tools/dockerfile/interoptest/grpc_interop_go/Dockerfile_1.7

```diff
@@ -0,0 +1,51 @@
+# Copyright 2015, Google Inc.
```

Comment:
filename should be just `Dockerfile`, that's a standard for docker.
grpc_interop_go1.7  or grpc_interop_go1_7 or sth in that sense